lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -59,6 +59,245 @@ static LogicalResult verifyLinalgCompatibleTypes(Operation *op,
   return success();
 }
 
+// Hack to deal with the Torch list type arguments which is not supported end
+// to end. Constant values can be be extracted directly and non constant
+// list values are not supported.
+// TODO: loose this constraint when properly support list type
+static bool constantIntListAsExpected(Value &value,
```

Comment:
can you give this a name that matches the coding standard? https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

I would call it `isConstantIntListMatching(Value &v, ArrayRef<int64_t> expected)`